### ISSUE TITLE
travis builds

### ISSUE THREAD

**Issue by jcvernaleo** [decred/decrediton#33]
We should be testing builds and PRs with travis like our other projects.

**Comment by jcvernaleo** [comment]
Can't really build this in travis (at least not without a complicated setup that I do not want) until there is a new grpc release that we can just npm install.  So I'm moving this to the future milestone since it is on hold while waiting on external factors.
